examples/training.py

```diff
@@ -30,6 +30,7 @@
 
 # Create both training and validation dataloaders
 train_dl = faster_rcnn.train_dataloader(train_ds, batch_size=16, num_workers=4, shuffle=True)
+
```

Comment:
Can you rollback this file? This update in unnecessary

Reply:
Yes rolled it back. Ctrl + Z is magic.